app/jobs/application_submission_job.rb

```diff
@@ -71,14 +71,16 @@ def find_ecf_user_by_email(user:)
       error_messages: ["#{e.class} - #{e.message}"],
       response_body:,
     )
-    Sentry.with_scope do |scope|
-      scope.set_context("User", { id: user.id })
-      Sentry.capture_exception(e)
+    unless e.is_a?(JsonApiClient::Errors::ConnectionError)
```

Comment:
I much more prefer `guard close` instead of wrapping all the things into `unless` block with can be quite misleading:

```
return if e.is_a?(JsonApiClient::Errors::ConnectionError)
```
what do you think?

Reply:
Yeah that's more easier to read.